merge.py

```diff
@@ -23,3 +31,39 @@ def merge_audio_video(combinations, video_files_path):
         # move/copy ? file
         shutil.move(audio_path, new_audio_path)
         shutil.move(video_path, new_video_path)
+
+
+def makedirs_if_needed(filename):
+    dirname = os.path.dirname(filename)
+    if not os.path.exists(dirname):
+        try:
+            os.makedirs(dirname)
+        except OSError as exc:  # Guard against race condition
+            if exc.errno != errno.EEXIST:
+                raise
+
+
+def move_files(dest_paths):
+    """
+    Move files to destination file locations
+    and create new folders if needed.
+    :param dest_paths: Dict of destination
+                       file locations.
+    :return: Nothing
+    """
+    for src_path, dest_path in tqdm(dest_paths.items(), desc='Moving', unit='file'):
```

Comment:
ako kluc v `dest_paths` su mena suborov. tie mena si getujes pomocou `os.walk`, ten by mal vraciat iba mena tych suborov a nie absolutnu cestu. cize nechapem ako potom funguje to `src_path`. mozes mi to opisat? dik

Reply:
aha. vidim ze v metode `get_record_filenames ` davas celu cestu. ale nazov premennej je filename. nebolo by dobre to premenovat na `absolute_file_path` alebo `file_path` napr. ?

Reply:
mozem upravit nazvy tak aby to bolo zjavnejsie a konzistentne
`filepath` = cesta (relativna alebo absolutna)
`filename` = nazov suboru (os.path.basename(filepath))